spec/features/consumer/cookies_spec.rb

```diff
@@ -0,0 +1,146 @@
+require 'spec_helper'
+
+feature "Cookies", js: true, retry: 3 do
+  include UIComponentHelper
+  include WebHelper
+
+  describe "policy link in the footer" do
+    scenario "showing" do
+      visit root_path
+      expect(page).to have_link "cookies policy"
+    end
+
+    scenario "opens cookies policy page" do
+      visit root_path
+      click_link "cookies policy"
+      expect_visible_cookies_policy_page
+    end
+  end
+
+  describe "related privacy policy link in the footer" do
+    scenario "not showing empty privacy policy link" do
+      Spree::Config[:privacy_policy_url] = nil
+      visit root_path
+      expect(page).to_not have_link "privacy policy"
+    end
+
+    scenario "showing configured privacy policy link" do
+      Spree::Config[:privacy_policy_url] = "link_to_privacy_policy"
+      visit root_path
+      expect(page).to have_link "privacy policy", :href => "link_to_privacy_policy"
+    end
+  end
+
+  describe "policy page" do
+    scenario "showing session_id cookies description with correct instance domain" do
+      visit '/#/policies/cookies'
+      expect(page).to have_content('_session_id')
+        .and have_content('127.0.0.1')
+    end
+
+    describe "with Matomo section configured" do
+      scenario "shows Matomo cookies details" do
+        Spree::Config[:cookies_policy_matomo_section] = true
+        visit '/#/policies/cookies'
+        # before { skip("test not working, BO config is not picked up") }
```

Comment:
I couldn't find the problem here. I don't think it's cache of the config setting. It could be page cache but I cant find anything on my /tmp/cache folder that indicates that.

I have removed BO from here.
BO is standard nomenclature for pretty much every software solution I've worked in the past, including e-commerce. FO means customer facing and BO means management UI. In OFN's case it's FO for customers and BO for enterprises. The super admin side can be included in the BO but it's not the central part.
I agree we should agree on nomenclature, I purpose a division that includes FO and BO here: https://community.openfoodnetwork.org/t/breaking-ofn-down-into-domains/1377